src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <title>To-Do list</title>
+</head>
+<body>
+    <div class="container" id="container">
+        <header>
+            <h3>Today's To Do</h3>
+            <a href ="#" type = "button" id="btn_head" class="btn btn-outline-secondary">
+                <i class="bi bi-arrow-repeat"></i>
+            </a>
+        </header>
+        <div class="input-group">
+            <input type="text" class="form-control" id="input_task" placeholder="Add to your list..." aria-label="Recipient's username" aria-describedby="button-addon2">
+            <button class="btn btn-outline-secondary" type="button" id="btn_input">
+                <i class="bi bi-arrow-return-left"></i>
+            </button>
+        </div>
+       
+        

```

Comment:
I think to make your code clean you could remove the empty spaces in the index.html file. You coule also use ctrl+shift+i to fix the indentation of the whole file in vscode.